packages/shared/src/components/cards/common/list/CardCover.tsx

```diff
@@ -13,11 +14,25 @@ export function CardCoverList({
   imageProps,
   ...props
 }: CardCoverProps): ReactElement {
+  const isLaptop = useViewSize(ViewSize.Laptop);
+  const optimizedImageProps = useMemo(() => {
+    const { src } = imageProps;
+    const mobileSrc = src.replace('/f_auto,q_auto/', '/t_mobile_feed/');
```

Comment:
Does t_ transformation already include the f_auto internally?

Reply:
They are not marked as "non next-gen format" from Lighthouse so I think it is included. 
I also tried to add the `f_auto` before, but is not possible to combine them!

Reply:
Ah cool yeah they apply it by default:
```
Optional. The file extension of the requested delivery format for the asset. Default: The originally uploaded format or the format determined by f_auto, when used.
```